src/components/expandable/expandable.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { ReactNode, useState } from 'react';
+import styled, { css } from 'styled-components';
+import { ReactComponent as ChevronSVG } from './expandable-chevron.svg';
+import { theme } from '../layout/theme';
+import { TextStyles } from '../typography/typography-v2';
+
+interface ExpandableProps {
+  /**
+   * Headline of the Expandable
+   * */
+  summary: string;
+  /**
+   * The actual text is passed as a child into the component
+   * */
+  children: ReactNode | ReactNode[];
+}
+
+interface ExpandableChevronProps {
+  open: boolean;
+  className?: string;
+}
+
+const ExpandableContainer = styled.details`
+  padding: 0 20px;
+`;
+
+const SummaryContainer = styled.summary`
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  color: ${theme.palette.text.topline};
+  list-style-type: none;
+
+  &::marker {
+    display: none;
+  }
+
+  &:hover {
+    color: ${theme.palette.text.default};
+  }
```

Comment:
Is it useful to use a vendor prefix here to make it temporarily usable in Safari as well?

```suggestion
  &::marker {
    display: none;
  }
  
  &::-webkit-details-marker{
    display: none;
  }

  &:hover {
    color: ${theme.palette.text.default};
  }
```

Reply:
Yes do it, it's a valid edge case in my opinion. Adding extra tooling for just this case is too much overhead.